### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by smisaacs**
Add some more clarity to install instructions. May cut down on errors leading to https://goo.gl/fsxSFi

### CODE REVIEW

**Comment by Andreyco** [comment]
Hey @smisaacs, thanks for contribution.
When you are installing (`yarn add react-native-payments-addon-{braintree|stripe}` it does install Carthage dependencies as well. Unfortunately, npm drops them on subsequent `npm install {package}` or `yarn add {package}`. 
This seems to be case for your PR, correct? Do you have any knowledge, how to prevent npm from doing this?

File: packages/react-native-payments-addon-stripe/README.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Second, install the [React Native Payments CLI](https://www.npmjs.com/package/re
 $ yarn add react-native-payments-cli
 ```
 
+Then, complete the Carthage dependency. Otherwise the next step fails:
+```bash
+$ cd node_modules/react-native-payments-addon-stripe
+$ carthage update
+```
+
```
Comment (Andreyco) [inline packages/react-native-payments-addon-stripe/README.md]: Could you please mention Braintree addon as well?

Comment (Andreyco) [reply packages/react-native-payments-addon-stripe/README.md]: And to be more precise, addon itself is working as expected.
The issue here is NPM auto pruning Carthage generated files. Change you propose fixes the issue until very next `npm install {package}` or `yarn add {package}`. We should mention this as well.

**Comment by nabilfreeman** [comment]
I'll close this issue as the library has been refactored to work with autolinking (0.60) (and CocoaPods for older 0.50x RN versions). Stripe and Braintree SDKs automatically link on iOS.

**Pull request closed without merging**